README.md

```diff
@@ -1,12 +1,26 @@
+---
+version: 0.8.0-SNAPSHOT
+---
+
+[![GitHub License](https://img.shields.io/github/license/devlinjunker/template.webpack.fend?color=blue)](https://github.com/devlinjunker/template.webpack.fend/blob/master/LICENSE)  
```

Comment:
- [ ] FOSSA https://bestpractices.coreinfrastructure.org/en/projects/new
- [ ] CII project needs to be created and badge re-added
```
[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fdevlinjunker%2Ftemplate.hapi.rest.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fdevlinjunker%2Ftemplate.hapi.rest?ref=badge_shield)

![![CII Best Practices Summary](https://img.shields.io/cii/summary/4287?label=core-infrastructure)](https://bestpractices.coreinfrastructure.org/en/projects/4287)
```